src/FINDOLOGIC/Api/Responses/Xml21/Properties/Filter/Item/CategoryItem.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace FINDOLOGIC\Api\Responses\Xml21\Properties\Filter\Item;
+
+use FINDOLOGIC\Api\Helpers\ResponseHelper;
+use FINDOLOGIC\Api\Responses\Xml21\Properties\Filter\CategoryFilter;
+use SimpleXMLElement;
+
+class CategoryItem extends Item
+{
+    public function __construct(SimpleXMLElement $item, CategoryFilter $filter)
+    {
+        parent::__construct($item);
+        $this->addSubItems($item, $filter);
+    }
+
+    /**
+     * Items might have sub items. Sub items may only be set for subcategories.
+     *
+     * @param SimpleXMLElement $response
+     * @param CategoryFilter $filter
+     */
+    private function addSubItems(
+        SimpleXMLElement $response,
+        CategoryFilter $filter
+    ) {
+        if (isset($response->items)) {
+            foreach ($response->items->children() as $item) {
+                $itemName = ResponseHelper::getStringProperty($item, 'name');
+                $this->items[$itemName] = Item::getInstance($filter, $item);
```

Comment:
The property `$items` should be declared in this class, as other items do not have sub-items.